src/core/internal/parseoptions.d

```diff
@@ -0,0 +1,275 @@
+/**
+* parse configuration options
+*
+* Copyright: Copyright Digital Mars 2017
+* License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+*/
```

Comment:
Maybe a short description on how the module is supposed to be used, with some example (that might be taken from the GC code) could be useful. At a first sight is not trivial to know how the API should be used.